### DISCUSSION TITLE
8월 31일 일기

### DISCUSSION THREAD

**Discussion by iamyanghee** [Melon-Shake/API#47] [team]
우와 오늘 8월 마지막 날이다

**Comment by jungssg** [comment]
집가고싶어요

**Comment by mews2000** [comment]
집가다 토할뻔
집가자마자 아무것도 못하고 옷만 벗고 디비 잠
그리고 명찰 잃어버림

**Comment by bksooon** [comment]
간만에 저녁에 너무 먹어서 아침부터 배가 아프다...

**Comment by json9101** [comment]
어제 무리를 해서 너무 피곤하다.  이제 D-21 끝까지 마무리 잘 했으면 좋겠다.

**Comment by iamyanghee** [comment]
윽...... 숙취.. 어서 정신 차리고 가야지..

**Comment by iamyanghee** [comment]
소피의 하루

- [x] 늦잠자고, 순대국과 아이스라떼로 해장하고, 2시부터 업무 시작함
- [x] sqlalchemy+pydantic 으로 client 저장하고 조회하는 부분까지만 코드와 파일 정리함 (결국 refresh_token 까지 받아오는 건 보류하기로 함. 일단 내 것만 고정으로 넣어둠.)
- [x] refresh_token 으로 access_token 받아오기
- [ ] sp_ 테이블 3종 적재해보기 (근데 route 에 있는 값이 얼마나 고유한지 고민하는 건 일단 제외하고)
